cocos/2d/components/deprecated.ts

```diff
@@ -23,81 +23,4 @@
  THE SOFTWARE.
  */
 
-/**
- * @packageDocumentation
- * @module ui
- */
-import { Mask } from './mask';
-import { Label } from './label';
-import { LabelOutline } from './label-outline';
-import { RichText } from './rich-text';
-import { Sprite } from './sprite';
-import { UIMeshRenderer } from './ui-mesh-renderer';
-import { Graphics } from './graphics';
-import { UIStaticBatch } from './ui-static-batch';
-import { UIOpacity } from './ui-opacity';
-import { js } from '../../core/utils/js';
-import { legacyCC } from '../../core/global-exports';
-
-/**
- * Alias of [[Mask]]
- * @deprecated Since v1.2
- */
-export { Mask as MaskComponent };
-legacyCC.MaskComponent = Mask;
-js.setClassAlias(Mask, 'cc.MaskComponent');
-/**
- * Alias of [[Label]]
- * @deprecated Since v1.2
- */
-export { Label as LabelComponent };
-legacyCC.LabelComponent = Label;
-js.setClassAlias(Label, 'cc.LabelComponent');
-/**
- * Alias of [[LabelOutline]]
- * @deprecated Since v1.2
- */
-export { LabelOutline as LabelOutlineComponent };
-legacyCC.LabelOutlineComponent = LabelOutline;
-js.setClassAlias(LabelOutline, 'cc.LabelOutlineComponent');
-
-/**
- * Alias of [[RichText]]
- * @deprecated Since v1.2
- */
-export { RichText as RichTextComponent };
-legacyCC.RichTextComponent = RichText;
-js.setClassAlias(RichText, 'cc.RichTextComponent');
-/**
- * Alias of [[Sprite]]
- * @deprecated Since v1.2
- */
-export { Sprite as SpriteComponent };
-legacyCC.SpriteComponent = Sprite;
-js.setClassAlias(Sprite, 'cc.SpriteComponent');
-/**
- * Alias of [[UIMeshRenderer]]
- * @deprecated Since v1.2
- */
-export { UIMeshRenderer as UIModelComponent };
-legacyCC.UIModelComponent = UIMeshRenderer;
-js.setClassAlias(UIMeshRenderer, 'cc.UIModelComponent');
-/**
- * Alias of [[Graphics]]
- * @deprecated Since v1.2
- */
-export { Graphics as GraphicsComponent };
-legacyCC.GraphicsComponent = Graphics;
-js.setClassAlias(Graphics, 'cc.GraphicsComponent');
-/**
- * Alias of [[UIStaticBatch]]
- * @deprecated Since v1.2
- */
-export { UIStaticBatch as UIStaticBatchComponent };
-js.setClassAlias(UIStaticBatch, 'cc.UIStaticBatchComponent');
-/**
- * Alias of [[UIOpacity]]
- * @deprecated Since v1.2
- */
-export { UIOpacity as UIOpacityComponent };
-js.setClassAlias(UIOpacity, 'cc.UIOpacityComponent');
+export {};
```

Comment:
I think it's better to remove this file directly

Reply:
I think you many need it when deprecated other API again, so just keep it. If we can make sure we will not use it to deprecate API again. We can delete it.

Reply:
As discussion with @pandamicro, will use deprecated-vxxx to record deprecated functions of a specific version, so i will remove these empty files.

Reply:
It is deleted.